### ISSUE TITLE
NumJitters causing problems

### ISSUE THREAD

**Issue by turowicz** [takuya-takeuchi/FaceRecognitionDotNet#31]
I get VERY different results between:

```
_faceRecognition.FaceEncodings(image, locations, 1);
```

and

```
_faceRecognition.FaceEncodings(image, locations, 100);
```

The first one gives values like 0.5324223232 but the other one gives values 3.954389834-E15, which is basically a 0.

**Comment by turowicz** [comment]
@takuya-takeuchi

**Comment by turowicz** [comment]
This works fine in python.

**Comment by turowicz** [comment]
I convert from BGR to RGB as the image input is opencv

**Comment by takuya-takeuchi** [comment]
@turowicz Thank you for your report!! Let me go through code.

**Comment by takuya-takeuchi** [comment]
@turowicz ok. I was succeed to reproduce issue. Please give me more time.

**Comment by takuya-takeuchi** [comment]
### Memo
There is problem in DlibDotNet.Dlib.Mean.

**Comment by takuya-takeuchi** [comment]
fix: 0143bd618aa9d918a6331f1409c8903db3645882
Need to build DlibDotNet.Native

**Comment by turowicz** [comment]
👍 thanks @takuya-takeuchi
